src/settings/BursarExports/BursarExportsConfiguration/BursarExportsConfiguration.test.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import user from '@testing-library/user-event';
+
+import '../../../../test/jest/__mocks__';
+import {
+  renderWithRouter
+} from '../../../../test/jest/helpers';
+
+import { BursarExportsConfiguration } from './BursarExportsConfiguration';
+import { SCHEDULE_PERIODS } from './constants';
+
+const defaultProps = {
+  onFormStateChanged: jest.fn(),
+  onSubmit: jest.fn(),
+};
+
+const renderBursarExportsConfiguration = (props = {}) => renderWithRouter(
+  <BursarExportsConfiguration {...defaultProps} {...props} />
+);
+
+describe('BursarExportsConfiguration', () => {
+  it('should render schedule period field', () => {
+    const { getByText } = renderBursarExportsConfiguration();
+
+    expect(getByText('ui-tenant-settings.settings.bursarExports.schedulePeriod')).toBeDefined();
+  });
+
+  it('should render ftp address field', () => {
+    const { getByText } = renderBursarExportsConfiguration();
+
+    expect(getByText('ui-tenant-settings.settings.bursarExports.ftpAddress')).toBeDefined();
```

Comment:
```suggestion
    expect(getByLabelText('ui-tenant-settings.settings.bursarExports.ftpAddress')).toBeVisible();
```

Reply:
`expect(...).toBeVisisble is not a function` + `getByLabelText` can't find label - no idea why

Reply:
`import '@testing-library/jest-dom'` for `toBeVisisble`